### PR TITLE
Update dist names in core docs 2.1.6 - closes #619

### LINKED ISSUE #619: Update distribution names in Core docs 2.1.6
All places in the docs, that mention the different distributions of Lisk Core:

Binary
Commander
Docker
Source

What should be the new content?

Application
Commander application
Docker image
Source code

Parent issue #589

### PR DESCRIPTION
**Pull request by chris529**
Closes #619 

Parent issue #589

### CODE REVIEW

File: modules/ROOT/pages/getting-started.adoc [hunk 1]
```diff
@@ -19,22 +19,22 @@ This is described on the xref:{url_core_distributions}[Lisk Core overview] page.
...
 
 The *default* way to set up Lisk Core.
-The xref:{url_binary_installation}[Binary installation] is an easy and automated way to set up Lisk Core, which includes almost completed automated update scripts; together with a selection of tools to assist with seamlessly maintaining a Lisk Node.
+The xref:{url_binary_installation}[Application installation] is an easy and automated way to set up Lisk Core, which includes almost completed automated update scripts; together with a selection of tools to assist with seamlessly maintaining a Lisk node.
 
-=== Commander
+=== Commander application
 
-Use xref:{url_lisk_commander}[Lisk Commander] to conveniently install and manage Lisk Core.
+Use xref:{url_lisk_commander}[Lisk Commander application] to conveniently install and manage Lisk Core.
```
Comment (Tschakki) [inline modules/ROOT/pages/getting-started.adoc]:
```suggestion
Use the xref:{url_lisk_commander}[Lisk Commander application] to conveniently install and manage Lisk Core.
```

File: modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -124,36 +123,36 @@ For further information, please visit the {url_explorer_testnet}[Lisk Testnet Ex
...
+The xref:{url_setup_binary}[Application] installation consists of a simple and mostly automated procedure to set up the Lisk Core.
 This includes almost entirely automated update scripts, coupled with a selection tools to help seamlessly maintain a Lisk node.
 --
 
-Lisk Commander::
+Lisk Commander application::
 +
 --
-Use xref:{url_setup_commander}[Lisk Commander] to conveniently install and manage Lisk Core.
+Use xref:{url_setup_commander}[Lisk Commander application] to conveniently install and manage Lisk Core.
```
Comment (Tschakki) [inline modules/ROOT/pages/index.adoc]:
```suggestion
Use the xref:{url_setup_commander}[Lisk Commander application] to conveniently install and manage Lisk Core.
```

File: modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -124,36 +123,36 @@ For further information, please visit the {url_explorer_testnet}[Lisk Testnet Ex
...
 --
-Docker::
+Docker image::
 +
 --
-xref:{url_setup_docker}[Docker] adds support for running a Lisk node with other platforms.
+xref:{url_setup_docker}[Docker image] adds support for running a Lisk node with other platforms.
```
Comment (Tschakki) [inline modules/ROOT/pages/index.adoc]:
```suggestion
The xref:{url_setup_docker}[Docker image] adds support for running a Lisk node with other platforms.
```

File: modules/ROOT/pages/interact-with-network.adoc [hunk 4]
```diff
@@ -33,7 +33,7 @@ It is highly recommended to utilise exchanges to xref:{url_maintain_node}[set up
 A delegate is also typically known as a *node operator*.
 . *Node operators and developers* also have a general interest in monitoring their node(s) as well as the network.
 Their nodes provide them with a private API that can be used to perform different queries, or to post transactions to the network.
-Depending on their preferences, node operators may wish to use <<javascript,Lisk Elements>>, <<commandline,Lisk Commander>> or a graphical interface such as <<lisk_desktop,Lisk Desktop>>.
+Depending on their preferences, node operators may wish to use <<javascript,Lisk Elements>>, <<commandline,Lisk Commander application>> or a graphical interface such as <<lisk_desktop,Lisk Desktop>>.
```
Comment (Tschakki) [inline modules/ROOT/pages/interact-with-network.adoc]:
```suggestion
Depending on their preferences, node operators may wish to use <<javascript,Lisk Elements>>, <<commandline,Lisk Commander>> or a graphical interface such as <<lisk_desktop,Lisk Desktop>>.
```

File: modules/ROOT/pages/interact-with-network.adoc [hunk 5]
```diff
@@ -64,8 +64,8 @@ One of the most useful packages in this regard is the xref:{url_elements_pkg_api
 [[commandline]]
 === B - Use the command-line
 
-xref:{url_commander}[Lisk Commander] is the CLI-tool that allows interaction with the network in a convenient manner through the command line.
-Please see the list of all commands and their example responses on the following: xref:{url_commander_commands}[Commands page].
+xref:{url_commander}[Lisk Commander application] is the CLI-tool that allows interaction with the network in a convenient manner through the command line.
```
Comment (Tschakki) [inline modules/ROOT/pages/interact-with-network.adoc]:
```suggestion
xref:{url_commander}[Lisk Commander] is the CLI-tool that allows interaction with the network in a convenient manner through the command line.
```

File: modules/ROOT/pages/management/commander.adoc [hunk 6]
```diff
@@ -1,18 +1,18 @@
...
-:page-previous-title: Commander installation
+:page-previous-title: Commander application installation
 
 :url_setup_commander: setup/commander.adoc
 :url_upgrade_commander: update/commander.adoc
 
 
 
-This section covers how to work with a Binary installation of Lisk Core.
+This section covers how to work with an application installation of Lisk Core.
```
Comment (Tschakki) [inline modules/ROOT/pages/management/commander.adoc]:
```suggestion
This section covers how to work with the commander application installation of Lisk Core.
```

File: modules/ROOT/pages/management/commander.adoc [hunk 7]
```diff
@@ -21,7 +21,7 @@ The available basic commands which can be used to manage your Lisk node are disp
 
 === Install
 
-NOTE: For more information, please see the xref:{url_setup_commander}[Setup page of Lisk Core with Lisk Commander].
+NOTE: For more information, please see the xref:{url_setup_commander}[Setup page of Lisk Core via Lisk Commander application].
```
Comment (Tschakki) [inline modules/ROOT/pages/management/commander.adoc]:
```suggestion
NOTE: For more information, please see the xref:{url_setup_commander}[Setup page of Lisk Core via Lisk Commander].
```

File: modules/ROOT/pages/setup/commander.adoc [hunk 8]
```diff
@@ -86,7 +86,7 @@ lisk core:status lisk-testnet
 --
 ====
 
-TIP: For all available options, see the Lisk Commander xref:{url_sdk_commander_liskcore}[Command reference for Lisk Core] as well as the xref:{url_sdk_commander_commands}[General command reference].
+TIP: For all available options, see the Lisk Commander application xref:{url_sdk_commander_liskcore}[command reference for Lisk Core] as well as the xref:{url_sdk_commander_commands}[General command reference].
```
Comment (Tschakki) [inline modules/ROOT/pages/setup/commander.adoc]:
```suggestion
TIP: For all available options, see the Lisk Commander xref:{url_sdk_commander_liskcore}[command reference for Lisk Core] as well as the xref:{url_sdk_commander_commands}[General command reference].
```

File: modules/ROOT/pages/setup/commander.adoc [hunk 9]
```diff
@@ -106,9 +106,9 @@ a|
 
 | `--release-url` | `-r`
 | `--release-url=https://my-custom.url/testnet/2.1.0-rc.0/lisk-2.1.0-rc.0-Linux-x86_64.tar.gz`
-| Define a custom URL for downloading the new Lisk Core binary script.
-It is also required to have the checksum for the TAR file with the format 2.0.0-Linux-x86_64.tar.gz.SHA256 for the Commander to check the integrity of the binary.
-You can refer to our {url_binary_download}[binary release] for the file naming convention.
+| Define a custom URL for downloading the new Lisk Core application script.
+It is also required to have the checksum for the TAR file with the format 2.0.0-Linux-x86_64.tar.gz.SHA256 for the Commander application to check the integrity of the application.
```
Comment (Tschakki) [inline modules/ROOT/pages/setup/commander.adoc]:
```suggestion
It is also required to have the checksum for the TAR file with the format 2.0.0-Linux-x86_64.tar.gz.SHA256 for the Commander to check the integrity of the application.
```

File: modules/ROOT/pages/update/application.adoc [hunk 10]
```diff
@@ -1,25 +1,25 @@
-= Update Binary
+= Update Application
 Mona Bärenfänger <mona@lightcurve.io>
-:description: The Binary update page describes how to upgrade Lisk Core to the latest version.
+:description: The Lisk Core aApplication update page describes how to upgrade Lisk Core to the latest version.
```
Comment (Tschakki) [inline modules/ROOT/pages/update/application.adoc]:
```suggestion
:description: The Lisk Core application update page describes how to upgrade Lisk Core to the latest version.
```

**Review by Tschakki — CHANGES_REQUESTED** [review]
Looking good, just a few minor comments.

**Comment by chris529** [comment]
Thanks Mona!

**Review by Tschakki — APPROVED** [review]
LGTM

**Pull request merged**